test/features/deploy_quarkus_service.feature

```diff
@@ -7,40 +7,27 @@ Feature: Deploy quarkus service
   Scenario Outline: Deploy drools-quarkus-example service without persistence
     Given Kogito Operator is deployed
     
-    When "<installer>" deploy quarkus example service "drools-quarkus-example" with native "<native>"
+    When Deploy quarkus example service "drools-quarkus-example" with native <native>
 
     Then Kogito application "drools-quarkus-example" has 1 pods running within <minutes> minutes
     And HTTP GET request on service "drools-quarkus-example" with path "persons/all" is successful within 2 minutes
-    
-    @cr
-    Examples: CR Non Native
-      | installer | native | minutes |
-      | CR        | false  | 10      |
-
-    @cr
```

Comment:
Is it OK that we no longer test deployments via CRs?

Reply:
I think the reason is to have it transparent, cli or cr, if there is no CLI set the cli option. is my understanding correct @radtriste ?

Reply:
by default, if nothing is given, it will try to use the CLI to deploy.
CLI is anyway creating a CR to deploy.

`cr_deployment_only` is there to say to the tests that we want only to test via CR (because ,for example, CLI is not available)

going to update the description